docs/guides/agent/agent_workflow/tool_components.md

```
@@ -10,558 +10,718 @@ sidebar_custom_props: {
 #### Configuration Example
 
-```json
-{
-  "content": "Generate a weekly report based on the workflow result.",
-  "output_format": "docx",
-  "filename": "weekly_report"
-}
-```
+| Configuration Item | Example Value |
+| --- | --- |
+| Content | `# RAGFlow Agent
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)